src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
+  display: flex;
+  align-items: center;
+
+  &:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b37;
+  transition: color 0.3s;
+
+  &:hover {
+    color: #00acdc;
+  }
+
+  &.is-active {
+    color: #00acdc;
+
+    &::after {
+      width: 100%;
+      display: flex;
+      content: "";
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      height: 4px;
+      border-radius: 2px;
+      background-color: #00acdc;
+    }
+  }
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s, color 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin: 0 3px 36px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    color: var(--Main-accent, #060b35);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    width: 166px;
+  }
+
+  &__product-code {
+    color: #616071;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__review-container {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-reviews-count {
+      color: #060b37;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+      color: #616071;
+    }
+
+    &-price {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: var(--text-accent-color);
+    }
+  }
+
+  &__buy {
+    margin-top: 16px;
+    width: 100%;
+    height: 40px;
+    display: flex;
+    border-radius: 5px;
+    background: var(--Blue-accent, #00acdc);
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    color: var(--White, #fff);
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+
+    &:hover {
+      color: var(--Blue-accent, #00acdc);
+      background: transparent;
+      border-color: #00acdc;
+    }
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &--1 .stars__star:nth-child(-n + 1),
+    &--2 .stars__star:nth-child(-n + 2),
+    &--3 .stars__star:nth-child(-n + 3),
+    &--4 .stars__star:nth-child(-n + 4),
+    &--5 .stars__star:nth-child(-n + 5) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
pay attention on checklist [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
use sass loop for stars block
![Screenshot from 2023-10-07 15-35-07](https://github.com/mate-academy/layout_catalog/assets/33029864/4033a6af-10a4-481a-b3a3-020a09e83067)